### PR TITLE
Disable UI scaling for in progress deployment

### PR DESCRIPTION
**Pull request by spadgett**
https://bugzilla.redhat.com/show_bug.cgi?id=1333129

@jwforres PTAL
@csrwng @kargakis CC

screen_shot_2016-05-05_at_12_48_08_pm

### CODE REVIEW

File: assets/app/scripts/controllers/deploymentConfig.js [hunk 1]
```diff
@@ -166,6 +166,8 @@ angular.module('openshiftConsole')
           }
 
           $scope.deployments = LabelFilter.getLabelSelector().select($scope.unfilteredDeployments);
+          $scope.deploymentInProgress = _.size($scope.deploymentConfigDeploymentsInProgress[deploymentConfigName]);
```
Comment (jwforres) [inline assets/app/scripts/controllers/deploymentConfig.js]: since you are treating this as a boolean can you `!!` it

**Comment by jwforres** [comment]
just the one comment otherwise LGTM

**Comment by spadgett** [comment]
Updated. Thanks @jwforres 

[merge]

**Comment by kargakis** [comment]
:+1:

**Comment by spadgett** [comment]
Rebased [merge]

**Pull request merged**